logback-android/src/main/java/ch/qos/logback/core/net/AbstractSocketAppender.java

```diff
@@ -186,7 +186,7 @@ protected void append(E event) {
     }
   }
 
-  private void connectSocketAndDispatchEvents() {
+  protected void connectSocketAndDispatchEvents() {
```

Comment:
Why are these scope changes necessary for this PR?

Reply:
It may have been an artifact of me using the alt-enter method to implement interfaces using intellij, will review any scope changes and try reverting them all.

Reply:
Looked more closely, I think there were some warnings or errors with tests previously. In any case, I've reverted these changes to scope throughout.